### PR TITLE
SUBMARINE-596. [WEB] Implement environment list through UI

### PR DESCRIPTION
**Pull request by kobe860219**
### What is this PR for?

* Implement get environment data from database.
* Show environment list through UI.

### What type of PR is it?
[Feature]

### What is the Jira issue?
https://issues.apache.org/jira/browse/SUBMARINE-596

### How should this be tested?
https://travis-ci.org/github/kobe860219/submarine/builds/717500078

### Screenshots (if appropriate)
截圖 2020-08-16 上午9 45 00

### Questions:
* Does the licenses files need update?No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by kobe860219** [comment]
@pingsutw  @liuxunorg  Please help me review this pr. Thanks a lot :))

File: submarine-workbench/workbench-web-ng/src/app/services/environment.service.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+import { Environment } from '@submarine/interfaces/environment-info';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class EnvironmentService {
+  constructor(private baseApi: BaseApiService, private httpClient: HttpClient) {}
+
+  fetchEnvironmentList(): Observable<Environment[]> {
+    const apiUrl = this.baseApi.getRestApi('/v1/environment/list');
```
Comment (pingsutw) [inline submarine-workbench/workbench-web-ng/src/app/services/environment.service.ts:35]: Thanks @kobe860219 
We use `api/v1/experiment` to list all experiments, so the `environment` API should consistent with `experiment`.
So we need to update #374 as well.
Rest LGTM

**Comment by pingsutw** [comment]
@kobe860219 I found that the environment ID is empty in the screenshot.

**Comment by kobe860219** [comment]
@pingsutw  I have removed the column of environment ID.

**Pull request closed without merging**